src/PolykeyClient.ts

```diff
@@ -34,51 +50,66 @@ interface PolykeyClient extends CreateDestroyStartStop {}
 )
 class PolykeyClient {
   static async createPolykeyClient({
-    nodePath = config.defaultsUser.nodePath,
-    streamFactory,
-    streamKeepAliveTimeoutTime,
-    parserBufferByteLimit,
+    port,
+    host,
+    options = {},
     fs = require('fs'),
     logger = new Logger(this.name),
     fresh = false,
   }: {
-    nodePath?: string;
-    streamFactory: StreamFactory;
-    streamKeepAliveTimeoutTime?: number;
-    parserBufferByteLimit?: number;
+    port: number;
+    host: string;
+    options: DeepPartial<PolykeyClientOptions>;
```

Comment:
Even when defaulted with `options = {},` above you still need to specifiy this asn optional with `?` otherwise the types will complain.

```suggestion
    options?: DeepPartial<PolykeyClientOptions>;
```